include/cantera/kinetics/RxnRates.h

```diff
@@ -621,8 +700,12 @@ class Chebyshev
     Array2D m_coeffs; //!<< coefficient array
     vector_fp chebCoeffs_; //!< Chebyshev coefficients, length nP * nT
     vector_fp dotProd_; //!< dot product of chebCoeffs with the reduced pressure polynomial
+
+    Units m_rate_units; //!< Reaction rate units
```

Comment:
I'd prefer to leave it with `Arrhenius`. The main reason is that for other rate types - e.g. `Falloff` - it is not immediately clear what the rate refers to. At the same time it is already handled by `Arrhenius`.

Reply:
That's all right for now, but I think the "rate units" still has a well-defined meaning for every reaction type because every reaction type ends up creating a value as returned by `getFwdRateConstants`, which must have some dimensions.

Reply:
Absolutely. The infrastructure is all there, but not access routine is implemented (yet). This probably should be handled by `ReactionRate::units`/`ReactionRate::setUnits` eventually, where `FalloffRate` handles things a little differently from the rest.